subprojects/packagefiles/glfw/tests/meson.build

```diff
@@ -0,0 +1,249 @@
+###  GLFW test programs
+#@ Build check
+if opt_tests == false
+  subdir_done()
+endif
+
+#@ Constants
+relpath = '../'
+incdirs = [include_directories(relpath + 'include')
+          ,include_directories(relpath + 'src')
```

Comment:
This relpath stuff is:
- a constant
- indirect, thus confusing
- leads to longer lines

I don't understand the goal.

Reply:
When initially writing and testing, I discovered that Meson was working from the current subdir (I had thought the working directory would remain at the level of the original meson.build). At the time, it was easier to add `relpath` to everything.